StylableSwiftUI/Classes/StylistIdentifierMatcher.swift

```diff
@@ -52,6 +71,13 @@ struct StylistIdentifierMatcher {
         var nextScore = 1<<(lhs.path.components.count * 2)// The score value of the current component - this goes up each time
         self.logger.debug("  Starting score delta \(nextScore)")
 
+        // The theme score value is 1 more level than the maximum score you could potentially have if the identifiers
+        // were identical. That way we guarantee that a match with a theme in any match will be higher than one without a theme
+        if rhs.theme != nil && theme != nil && rhs.theme == theme {
+            let themeScoreValue = theme == nil ? 0 : 1<<((lhs.path.components.count + 1) * 2)
+            score += themeScoreValue
+        }
+
```

Comment:
Just to confirm, this means that the style for `dark@title` will override `section/element/button/title` _and_ `section/element/label/title` - is that what we want, because these probably won't have the same styling (i.e. font size) before a theme is applied? Or are we just not expecting this to be how the library is used? 

It might be exactly what we want, this is a genuine question!

Reply:
That's correct. 

@cdemetriadis and I talked about this and agreed that anything in the should have a higher precedence than anything without a theme, regardless of the path to the token.